doc/tasks/kahlan.md

```diff
@@ -0,0 +1,30 @@
+# Phpspec
+
+Kahlan is a full-featured Unit & BDD test framework a la RSpec/JSpec which uses a describe-it syntax and moves testing in PHP one step forward.
+It lives under the `kahlan` namespace and has following configurable parameters:
+
+```yaml
+# grumphp.yml
+parameters:
+    tasks:
+        kahlan:
+          config: ~
+          src: ~
+          spec: ~
+          pattern: ~
+          reporter: ~
+          coverage: ~
+          clover: ~
+          istanbul: ~
+          lcov: ~
+          ff: ~
+          'no-colors': ~
+          'no-header': ~
+          include: ~
+          exclude: ~
+          persistent: ~
+          cc: ~
+          autoclear: ~
+```
+
+Every options available are documented on [kahlan](https://kahlan.github.io/docs/cli-options.html)
```

Comment:
Can you add more information about the options in this document? 
In the other tasks, we describe every possible option.
We would like to keep the documentation in the same format as the other tasks.